### PR TITLE
fix several typos and misleading wordings

### PR DESCRIPTION
**Pull request by kaeku**

### CODE REVIEW

File: _posts/2016-02-22-angular-2-change-detection-explained.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ Let's assume that somewhere in our component tree an event is fired, maybe a but
 
 Since each component has its own change detector, and an Angular application consists of a component tree, the logical result is that we're having a **change detector tree** too. This tree can also be viewed as a directed graph where data always flows from top to bottom.
 
-The reason why data flows from top to bottom, is because change detection is also always performed from top to bottom for every single component, every single time, starting from the root component. This is awesome, as unidirectional data flow is more predictable than cycles. We always know where the data we use in our views comes from, because it can only come from its component.
+The reason why data flows from top to bottom, is because change detection is also always performed from top to bottom for every single component, every single time, starting from the root component. This is awesome, as unidirectional data flow is more predictable than cycles. We always know where the data we use in our views comes from, because it can only result from its component.
```
Comment (PascalPrecht) [inline _posts/2016-02-22-angular-2-change-detection-explained.md]: This change I'm not sure about

**Comment by PascalPrecht** [comment]
LGTM, left a comment

**Comment by PascalPrecht** [comment]
Thanks!

File: _posts/2016-02-22-angular-2-change-detection-explained.md (hunk 1 above)
Comment (kaeku) [inline _posts/2016-02-22-angular-2-change-detection-explained.md]: Felt like the sentence was a bit bumpy.. Not 100% sure about it though.

**Comment by kaeku** [comment]
Welcome!

**Pull request merged**